src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -103,6 +110,33 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid new_buckets_path (leading space in new_buckets_path)', async () => {
+            const { type, name, uid, gid, new_buckets_path } = defaults;
+            const account_options = { config_root, name, uid, gid}; // will add new_buckets_path with leading space later
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const res = await exec_manage_cli_add_leading_space_option(command, 'new_buckets_path', new_buckets_path, quoted_type.SPACE_ONLY); // only space
```

Comment:
Now, you don't need the comment (it is clear from the enum) - you can leave it or delete it, whatever you think.
```suggestion
            const res = await exec_manage_cli_add_leading_space_option(command, 'new_buckets_path', new_buckets_path, quoted_type.SPACE_ONLY);
```

Reply:
Sure, Thanks.